Bursts/matlab/getBurstOriginXYN.m

```diff
@@ -1,40 +1,51 @@
-%GETBURSTORIGINXYN return burst origin  (X,Y) and neuron number N
-%Read <binnedBurstInfo.csv> and retrieve burst x,y origin by calculating 
-%the centroid location of neurons that spiked the most in the first 100 
-%timesteps for each burst (brightest pixel in the starting frame)
+% GETBURSTORIGINXYN return burst origin  (X,Y) and neuron number N
+% Read <binnedBurstInfo.csv> and retrieve burst x,y origin by calculating 
+% the centroid location of neurons that spiked the most in the first 100 
+% timesteps for each burst (brightest pixel in the starting frame)
 %
-%   Syntax: getBurstOriginXY(h5file)
+%   Syntax: getBurstOriginXY(h5dir)
```

Comment:
Fix name.

Reply:
Also add in return values, like other functions.

Reply:
fixed and added return values